prowler/providers/aws/services/ecs/ecs_container_not_directly_publicly_accessible_via_elbv2/ecs_container_not_directly_publicly_accessible_via_elbv2.py

```diff
@@ -0,0 +1,30 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.ecs.ecs_client import ecs_client
+from prowler.providers.aws.services.elbv2.elbv2_client import elbv2_client
+
+
+class ecs_container_not_directly_publicly_accessible_via_elbv2(Check):
+    def execute(self):
+        findings = []
+
+        public_containers = {}
+        for tg in elbv2_client.target_groups:
+            if tg.public and tg.target_type == "ip":
+                public_containers[tg.target] = tg.arn
```

Comment:
You should check the firewall (Security Group) of the container before adding it to public containers to ensure that is really public through Internet. This is not added yet in the ECS service, so if you need help to add it please let me know and I can help you adding it to your PR as soon as possible.

Reply:
Yeah sure, any help would be greatly appreciated